src/main/java/com/microsoft/azure/datalake/store/ADLStoreClient.java

```diff
@@ -228,6 +232,22 @@ public static ADLStoreClient createClient(String accountFQDN, MsiTokenProvider t
         return createClient(accountFQDN, (AccessTokenProvider) tokenProvider);    }
 
 
+	/**
+	 * Set parameters for exponential backoff
+	 * @param maxRetries Maximum number of retry attempts
+	 * @param exponentialRetryInterval Initial retry interval, in milliseconds
+	 * @param exponentialFactor Retry interval is geometrically increased by this factor on each attempt
+	 */
+	public void setExponentialBackoffParameters(int maxRetries, int exponentialRetryInterval, int exponentialFactor) {
+		this.maxRetries = maxRetries;
+		this.exponentialRetryInterval = exponentialRetryInterval;
+		this.exponentialFactor = exponentialFactor;
+	}
+	
+	public ExponentialBackoffPolicy makeExponentialBackoffPolicy() {
```

Comment:
make it internal method

Reply:
@rahuldutta90 , I don't quite follow your comment.  Should this still be a method of the client or should it just move somewhere else -- ExponentialBackoffPolicy?  Which would need access to the options.

Reply:
@wheezil  yes this can still be a method on client but just make it a internal method (not public) since you are accesing it only within this adlstoreclient

Reply:
I've pushed a new commit.  The access is "package private" since other classes need it.